solr/core/src/test/org/apache/solr/cli/PostToolTest.java

```diff
@@ -281,15 +281,15 @@ public void testAppendUrlPath() {
   @Test
   public void testGuessType() {
     File f = new File("foo.doc");
```

Comment:
total nit pick, but would declaring the `File f` as a `Path p` save a `.toPath`???

Reply:
Yes - but I thought it was best to update the test files separately and at a later point.

Reply:
awesome!